### ISSUE TITLE
Fails unless TZ=America/Los_Angeles (0.01_02-TRIAL)

### ISSUE THREAD

**Issue by eserte** [barefootcoder/date-easy#6]
t/format.t fails for most timezones. E.g. for TZ=UTC:
```
#   Failed test 'strftime format: %s'
#   at t/format.t line 26.
#          got: '981173106'
#     expected: '981201906'
# Looks like you failed 1 test of 4.
t/format.t .............
Dubious, test returned 1 (wstat 256, 0x100)
Failed 1/4 subtests
```

**Comment by barefootcoder** [comment]
Man, I hate timezones. :-/

Okay, this one is caused by two things.  First, I forgot to force my test datetime to UTC.  But, secondly, that doesn't fix it, because there is a weirdness on the Time::Piece side (see rjbs/Time-Piece#24).

So I think the right answer is to do the workaround I suggest for Time::Piece in that issue here in Date::Easy.  That way we'll always get the right answer regardless of what Time::Piece does about `%s`.

Thx for another clear error report.  I'll try to put together a fix for this tomorrow.

**Comment by barefootcoder** [comment]
Fixed.  Will close once CPAN Testers verifies.

**Comment by barefootcoder** [comment]
Should be fixed in 0.02.
